### PR TITLE
Add top nav bar for fencing mode

### PR DESCRIPTION
**Pull request by JohnHKoh**
Just wrapped the entire fencing element around a `<Flex>` and added a `<Nav>` component.
image

Address "Banner at the top that goes back to the main site" from #172.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Handle line endings automatically for files detected as text
```
Comment (stevenhao) [inline .gitattributes:1]: could you explain this change a bit? i'm not fully familiar with gitattributes. also perhaps it could be a separate PR

Comment (JohnHKoh) [reply .gitattributes:1]: Shoot sorry, I didn’t mean to commit that. Since I’m working in Windows I needed to make sure my line endings were consistent with the rest of the repo. I’ll take it out and update this PR later.

Comment (JohnHKoh) [reply .gitattributes]: Should be good now.
